tools/Ecoregionalization_workflow/cluster.xml

```diff
@@ -0,0 +1,132 @@
+<tool id="Clara_cluster" name="ClaraClust" version="0.1.0+galaxy0" profile="22.05">
+    <description>Make cluster from BRT prediction with Clara algorithm</description>
+    <requirements>
+       <requirement type="package" version="4.2.3">r-base</requirement>
+       <requirement type="package" version="2.1.4">r-cluster</requirement>
+       <requirement type="package" version="1.1.1">r-dplyr</requirement>
+       <requirement type="package" version="2.0.0">r-tidyverse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/cluster_ceamarc.R'
+         '$input1'
+         '$input2'
+         '$input3'
+         '$k'
+         '$metric'
+         '$sample'
+         '$output1'
+         '$output2'
+         '$output3'
+    ]]></command>
+    <inputs>
+      <param name="input1" type="data" format="tabular" label="Prediction matrix (data to cluster from previous step) "/>
+      <param name="input2" type="data" format="txt,csv,tabular" label="Environment file"/>
+      <param name="input3" type="data" format="tabular" label="Prediction file (data.bio table from previous step)"/>
```

Comment:
what is the previous step?

Reply:
Cluster estimate, modified! THANK YOU, I LOVE YOU !!!